tests/default_vars.sh

```diff
@@ -826,8 +879,15 @@ export CDMBWD_c192='0.23,1.5,1.0,1.0'
 export CDMBWD_c384='1.1,0.72,1.0,1.0'
 export CDMBWD_c768='4.0,0.15,1.0,1.0'
 
+#DT_INNER=(Time step)/2
+export DT_INNER_c96=350
```

Comment:
Should this be 360? If the time step is 700s, we won't get hourly output if the time step is 700.

Reply:
Actually it looks like the time step is 720 (https://github.com/JessicaMeixner-NOAA/ufs-weather-model/blob/feature/p8b_aero/tests/default_vars.sh#L1101) so I did not do 720/2 correctly and this should be 360 (or I found a dt=700 somewhere and now am not seeing it).  Eitherway, I'm updating this to 360.